Userland/Libraries/LibGUI/TextEditor.cpp

```diff
@@ -105,6 +106,7 @@ void TextEditor::create_actions()
             this);
     }
     m_select_all_action = CommonActions::make_select_all_action([this](auto&) { select_all(); }, this);
+    m_insert_emoji_action = CommonActions::make_insert_emoji_action([&](auto&) { insert_emoji(); }, this);
```

Comment:
I think this action needs to be disabled in `TextEditor::set_mode`. We toggle `set_accepts_emoji_input` in there already.

Reply:
Good catch!